### PR TITLE
[basicprofiles] Add a CHAIN profile

### PR DESCRIPTION
**Pull request by J-N-K**
This profile allows to chain transformations, similar to what MQTT or HTTP offer in the state and command transformation configurations.

### CODE REVIEW

File: bundles/org.openhab.transform.basicprofiles/README.md [hunk 1]
```diff
@@ -208,3 +208,74 @@ Number:Temperature airconTemperature{
...
+
+```
+locked=ON
+unlocked=OFF
+```
+
+So the full `toItem` transformation is
+
+```
+JSONPATH:$.device.status.temperature∩MAP:doorlock.map
```
Comment (jlaur) [inline bundles/org.openhab.transform.basicprofiles/README.md:249]: Shouldn't it be:
```suggestion
JSONPATH:$.device.status.lock∩MAP:doorlock.map
```

?

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks! Only one comment for the documentation.

**Comment by jimtng** [comment]
Should `toChannel` be called `toHandler`, to be consistent with the Script Profile and also [Debounce time profile](https://www.openhab.org/addons/transformations/basicprofiles/#debounce-time-profile)?

**Comment by J-N-K** [comment]
For the script profile we have `toItemScript`, `commandFromItemScript`, `stateFromItemScript`. So also different.

**Comment by jimtng** [comment]
Maybe we can adopt `FromItem` and change the Debounce time profile, i.e. support the old `toHandlerDelay` with a deprecation warning, and add `fromItemDelay`?

**Comment by ccutrer** [comment]
Is there a reason this is a profile, and not another transformation? Seems useful to be able to chain transformations anywhere a transformation can be used. And it should automatically be usable as a profile, no, since all transformations can be used as profiles?

**Comment by lsiepel** [comment]
Ping @J-N-K please fox the conflict ? Also have  similar question as @ccutrer.

**Comment by J-N-K** [comment]
I currently don't have time to work on this one. If someone wants implement that, I'm fine with it.

**Pull request closed without merging**